lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2259,7 +2263,7 @@ export class UserAgentApplication {
    * @ignore
    * @hidden
    */
-  private getPostLogoutRedirectUri(): string {
+  public getPostLogoutRedirectUri(): string {
```

Comment:
why are these public

Reply:
Mentioned above: "This is user i/p which we need for some Unit testing since many variables are unpacked in later APIs in UserAgentApplication. I think that is the reasoning for this particular change too."

Reply:
These functions are required to be public to check the state. These are variables set by the user.

Reply:
As we break up UAA, we should look to design in ways that let us test without exposing surface area, but this is fine for now.